hack/generator/pkg/astbuilder/calls.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astbuilder
+
+import "go/ast"
+
+// CallFunc() creates an expression to call a function with specified arguments
+func CallFunc(funcName *ast.Ident, arguments ...ast.Expr) ast.Expr {
```

Comment:
Include samples of what the call looks like in the comments (as the other methods do?). Won't repeat this but it applies to a bunch of the new methods here.